codewof/static/js/question_types/debugging.js

```diff
@@ -152,3 +158,56 @@ function run_python_code(user_code, test_case) {
         test_case.runtime_error = true;
     }
 }
+
+
+function setTutorialAttributes() {
+    $(".question-text").attr(
+        'data-intro',
+        'This is a description of what the code should do. In debugging questions there will be one or more bugs in the given code that you have to find and correct in order for all the tests to pass.'
+    );
+    $("#python-editor").attr(
+        'data-intro',
+        "This is the code that contains the bug(s). Find the bug(s) and correct them. The greyed out lines cannot be edited."
+    );
+    $("#run_code").attr(
+        'data-intro',
+        "Click 'Run code' to run your code against the test cases and see if you fixed the bug!"
```

Comment:
Maybe reword this so it isn't an instruction - people might actually click the button and get confused when nothing happens